addons/binding/org.openhab.binding.meteostick/README.md

```diff
@@ -19,6 +19,15 @@ The Meteostick things need to be manually added - there is no discovery in the M
 First add and configure the Meteostick bridge - the port and frequency band for your region need to be set.
 Next add the sensor and configure the channel number.
 
+### Thing Configuration
+
+Things can be defined in the .thing file as follows
+
+```
+meteostick:meteostick_bridge:receiver [ port="/dev/tty.usbserial-AI02XA60" mode="1" ]
```

Comment:
This part should rather go into the "Full Example" section: https://github.com/eclipse/smarthome/blob/master/tools/archetype/binding/src/main/resources/archetype-resources/README.md#full-example